### DISCUSSION TITLE
How can I remove a solid from a compound shape?

### DISCUSSION THREAD

**Discussion by kriho** [sgenoud/replicad#151] [Q&A]
Given a compound shape, and a list of its solids, is it possible to remove some of the solids from that shape without using boolean operations?

**Comment by sgenoud** [comment]
If you have a list of solids that are part of the new compound, you can create a new compound out of these solids.

**Reply by kriho** [reply]
Sounds like there is no way to remove a solid from a compound. How can I create a compound?
